system-test/integration_test.go

```diff
@@ -93,7 +93,7 @@ retry git fetch origin {{if .PR}}pull/{{.PR}}/head{{else}}{{.Branch}}{{end}}:pul
 git checkout pull_branch
 git reset --hard {{.Commit}}
 
-retry npm install --nodedir="$NODEDIR" &>/dev/ttyS2
+retry node $HOME/cloud-profiler-nodejs/system-test/npm-install-retry.js 60000 3 install --nodedir="$NODEDIR" &>/dev/ttyS2
```

Comment:
Do you truly mean to retry the already retrying command?

Reply:
Good catch!
Switched to using the approach you'd suggested that uses timeout.